test/controllers/api/v1/developments_controller_test.rb

```diff
@@ -37,6 +43,12 @@ def development
     end
   end
 
+  test 'should handle empty params with grace and poise' do
```

Comment:
I just realized one test needed is multiple empty filters. This seems to return nothing.